src/xtgeo/common/calc.py

```diff
@@ -320,38 +359,34 @@ def vectorpair_angle3d(p0, p1, p2, degrees=True, birdview=False):
         ValueError: Errors in input dimensions, all points must have 3 values
     """
 
-    p0 = np.array(p0)
-    p1 = np.array(p1)
-    p2 = np.array(p2)
+    _p0 = np.array(p0)
+    _p1 = np.array(p1)
+    _p2 = np.array(p2)
 
-    if p0.size != p1.size or p0.size != p2.size or p0.size != 3:
+    if _p0.size != 3 or _p1.size != 3 or _p2.size != 3:
```

Comment:
`if any((_p0.size, _p1.size, _p2.size)) != 3:` would be a more Pythonic way of doing this

Reply:
Agree :-)